include/zephyr/bluetooth/cs.h

```diff
@@ -620,6 +620,136 @@ int bt_le_cs_create_config(struct bt_conn *conn, struct bt_le_cs_create_config_p
  */
 int bt_le_cs_remove_config(struct bt_conn *conn, uint8_t config_id);
 
+/** @brief CS Security Enable
+ *
+ * This commmand is used to start or restart the Channel Sounding Security
+ * Start procedure in the local Controller for the ACL connection identified
+ * in the conn parameter.
+ *
+ * @note To use this API @kconfig{CONFIG_BT_CHANNEL_SOUNDING} must be set.
+ *
+ * @param conn   Connection Object.
+ *
+ * @return Zero on success or (negative) error code on failure.
+ */
+int bt_le_cs_security_enable(struct bt_conn *conn);
+
+struct bt_le_cs_procedure_enable_param {
+	uint8_t config_id;
+	uint8_t enable;
+};
+
+/** @brief CS Procedure Enable
+ *
+ *  This command is used to enable or disable the scheduling of CS procedures
+ *  by the local Controller, with the remote device identified in the conn
+ *  parameter.
+ *
+ * @note To use this API @kconfig{CONFIG_BT_CHANNEL_SOUNDING} must be set.
+ *
+ * @param conn   Connection Object.
+ * @param params Parameters for the CS Procedure Enable command.
+ *
+ * @return Zero on success or (negative) error code on failure.
+ */
+int bt_le_cs_procedure_enable(struct bt_conn *conn,
+			const struct bt_le_cs_procedure_enable_param *params);
+
+struct bt_le_cs_set_procedure_parameters_param {
+	uint8_t config_id;
+	uint16_t max_procedure_len;
+	uint16_t min_procedure_interval;
+	uint16_t max_procedure_interval;
+	uint16_t max_procedure_count;
+	uint32_t min_subevent_len;
+	uint32_t max_subevent_len;
+	uint8_t tone_antenna_config_selection;
```

Comment:
I think using enums helps prevent users from using wrong values

For 
- tone_antenna_config_selection
- snr_control_initiator
- snr_control_reflector

you can re-use the existing enums in this file. It looks like they just need to be renamed to remove the word "test"